SUI/SUI/SignUpView.swift

```diff
@@ -0,0 +1,287 @@
+        static let gradientViewHeight = 70
+        static let singInToggleCapsuleFrameWidth = 300.0
+        static let singInToggleCapsuleFrameHeight = 80.0
+        static let singInToggleCapsuleOpacity = 0.25
+        static let singInToggleCircleFrameWidth = 72.0
+        static let singInToggleCircleOffsetX = -109.0
+        static let singInToggleRectangleFrameWidth = 110.0
+        static let singInToggleRectangleFrameHeight = 72.0
+        static let singInToggleRectangleOffsetX = -55.0
+        static let singInToggleLeftText = "Log in"
+        static let singInToggleLeftTextFontSize = 25.0
+        static let singInToggleLeftTextOpacity = 0.5
+        static let singInToggleLeftTextOffsetX = -70.0
+        static let singInToggleRightText = "Sing up"
+        static let singInToggleRightTextFontSize = 25.0
+        static let singInToggleRightTextOffsetX = 70.0
+        static let loginTextFieldPlaceholder = "+0 (000) 000-00-00"
+        static let loginTextFieldMaxCharactersCount = 18
+        static let passwordTextField = "Password"
+        static let passwordPlaceholderText = "● ● ● ● ● ● ●"
+        static let rectangleHeight = 1
+        static let rectanglePaddingLeading = 12
+        static let rectanglePaddingTrailing = 12
+        static let rectanglePaddingTop = 10
+        static let passwordMaximumCount = 15
+        static let passwordMinimumCount = 6
+        static let signUpButtonLabel = "SIGN UP"
+        static let signUpButtonWidth = 200
+        static let signUpButtonHeight = 50
+        static let signUpButtonCornerRadius = 25
+        static let font = 20
+        static let forgotPasswordButtonLabel = "Forgot your password?"
+        static let techSupportNumber = "Tech support number 8800 555 35 35"
+        static let verificationButtonLabel = "Verification"
+    }
+
+    // MARK: - Public Properties
+
+    var body: some View {
+        VStack(spacing: CGFloat(Constants.vStackSpacing)) {
+            topGradientView
+            Spacer()
+            logInSignInView
+            Spacer()
+            loginTextView
+            passwordTextView
+            signUpButtonView
+            forgotPasswordButtonView
+            verificationButtonView
+        }
+        .padding(.bottom, CGFloat(Constants.bottomPadding))
+        .onTapGesture {
+            hideKeyboard()
+        }
+    }
+
+    // MARK: - Private Properties
+
+    @FocusState private var loginIsFocused: Bool
+    @FocusState private var passwordIsFocused: Bool
+
+    private var phoneNumberFormatter = PhoneNumberFormatter()
+
+    @StateObject private var viewModel = SignUpViewModel()
+
+    private var topGradientView: some View {
+        VStack {
+            EmptyView()
+        }
+        .frame(
+            width: UIScreen.main.bounds.width,
+            height: CGFloat(Constants.gradientViewHeight)
+        )
+        .background(
+            LinearGradient(
+                colors: [.yellow, .orange],
```

Comment:
```suggestion
                colors: [.yellow, 
                .orange],
```
обычно после запятой ставят энтер, но как сам считаешь нужным